index.js

```diff
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

Comment:
Huh, didn't know that.

Ok, I can put in a check to see if the debugging is enabled before registering. That work?